index.js

```diff
@@ -1,88 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+var counter = 1
+console.log("%cQuestion 1", "color: red") 
+console.log(counter) 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+
+// => 2
+var counter = 2
+console.log(counter)
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
 
-// console.log(name)
+const name = 'Henry Velasquez'
+console.log(name)   // WHY DOES NAME HAVE A STRIKETHROUGH?
 // => "Raffy"
-// name = "Not Raffy" 
+
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
+function drinkWater(currentThirstLevel) { 
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -= 1
+  console.log( 'Ahh that hits the spot' )
+  return currentThirstLevel
+}
+ console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num, maybeNum) {
```

Comment:
Nice! I also think it's worth exploring the difference between `===` and `==`. Another solution could look like:

```javascript
function sameSameButDifferent(num, maybeNum) {
  if (num === maybeNum) {
    return "same same"
  }
  else if (num == maybeNum) {
    return "same same (but different)"
  }
  else {
    return "different"
  }
}
```